lib/components/user/nav-login-button.js

```diff
@@ -99,7 +102,12 @@ export default class NavLoginButton extends Component {
           <MenuItem header>{displayedName}</MenuItem>
 
           {links && links.map((item, index) => (
-            <MenuItem href={item.url} key={index} target={item.target}>{item.text}</MenuItem>
+            <MenuItem
+              onSelect={() => this.props.routeTo(item.url)}
```

Comment:
Make this onSelect a class method so that a new function isn't created every render.